modules/ROOT/pages/about-designing-a-mule-application.adoc

```diff
@@ -4,10 +4,52 @@ include::_attributes.adoc[]
+
+In Flow Designer, you create Mule applications as projects having one or more flows. A project contains the flows for one Mule application. A flow consists of a sequence of _cards_, with each card representing a connector, core Mule Runtime component, API, or module. Each card receives an input, carries out a specific task, and then sends the output of that task to the next card in the flow. The final card in a flow is usually one that sends a notification that the Mule application has completed its task or that sends data.
+
+== Getting Started
+
+To learn about Flow Designer by building an example Mule application, you can try the tutorial xref:salesforce-to-twilio.adoc[Create Your First Mule Application with Flow Designer]. The tutorial guides you in creating a Mule app that places bulk orders for company swag automatically when opportunities are closed, and that notifies customers via SMS about their order status and ETA.
+
+To start creating Mule applications:
+
+* Try the easiest option, which is to import a template into Flow Designer from Anypoint Exchange. See xref:import-templat.adoc[Create a Mule Application Quickly from a Template].
+* Create a new Mule-app project in Design Center and use the wizard that automatically opens to walk you through scaffolding a flow with a trigger card and a target card.
+* Create a new Mule-app project and click *Go straight to the canvas* in the wizard to start building a flow from a blank canvas.
+
+== Tour of the Flow Designer Interface
+
+.The five areas of the Flow Designer interface
+image::FD-annotated.png[]
+
+. The canvas
++
+Here you can create a flow. The cards in a flow appear in the middle, in a sequence from left to right. The first card in a flow is the trigger card, which represents the connector and operation that you want to use to trigger the flow.
++
+In the top-left corner of the canvas are the names of the project and the flow.
+. The panes
++
+Click the top icon to open the *Project* pane, which lists a project's flows, the configurations that are being used, and any global data types that have been defined.
++
+Click the bottom icon to open the *Dependencies* pane, which lists a project's dependencies.
++
+What are reusable configurations?:: When you add a card that connects to another system, you specify how Flow Designer can locate and authenticate to that system. For example, if you add a Database card to a flow, you need to specify the type of database, its host, other connectivity information, and how to authenticate. The set of this information is called a _configuration_. After you add one Database card, the next Database card you add uses the same configuration by default, though you can change it. In the Project pane, the list of resuable configurations is the list of all of the configurations that you have created in a project and that can be used in additional cards that connect to the same systems as cards that are already in the project.
```

Comment:
I would break the last sentence into two sentences for clarity.

Reply:
Instead, I removed "that connect to the same systems as cards that are already in the project".